### PR TITLE
Introduced djangoCMS 3.4 compatibility

### PR DESCRIPTION
**Pull request by czpython**

### CODE REVIEW

File: djangocms_text_ckeditor/forms.py [hunk 1]
```diff
@@ -43,7 +43,8 @@ def get_child_plugins(self):
     def render_plugin(self, request):
         plugin = self.cleaned_data['plugin']
         context = RequestContext(request)
-        rendered_content = plugin.render_plugin(context)
+        context['request'] = request
```
Comment (Chive) [inline djangocms_text_ckeditor/forms.py]: is that really needed o_O

File: djangocms_text_ckeditor/models.py [hunk 2]
```diff
@@ -100,59 +102,6 @@ def post_copy(self, old_instance, ziplist):
...
-
-                    # Save changes to parent text plugin
-                    text = '<img alt="%s" id="plugin_obj_%s" src="%s" title="%s">' % (match[2], match[1], match[4], match[3])
-                    value = value.replace(match[0], text)
-
-            setattr(self, field, value)
-        self.save()
-
-        return True
-
```
Comment (Chive) [inline djangocms_text_ckeditor/models.py]: 😢

**Comment by Chive** [comment]
👍

File: djangocms_text_ckeditor/forms.py (hunk 1 above)
Comment (czpython) [inline djangocms_text_ckeditor/forms.py]: Django does not apply context processors until the template is rendered.
This means that I can't access context['request'] prematurely and so I need to set it explicitly

File: djangocms_text_ckeditor/tests/test_plugin.py [hunk 3]
```diff
@@ -138,18 +138,21 @@ def test_add_and_edit_plugin(self):
...
 
             self.assertEqual(response.status_code, 200)
 
             # Assert cancel token is present
             self.assertContains(response, action_token)
 
-        request = self.get_post_request({'body': "Hello world"})
-        response = native_placeholder_admin.edit_plugin(request, text_plugin_pk)
+        with self.login_user_context(self.get_superuser()):
+            data = {'body': "Hello world"}
```
Comment (Chive) [inline djangocms_text_ckeditor/tests/test_plugin.py]: two different quotes style on one line. Impressed

Comment (czpython) [inline djangocms_text_ckeditor/tests/test_plugin.py]: I know right! crazy stuff 👯

**Pull request merged**